TCAT/Model/BusLocation.swift

```diff
@@ -28,6 +28,8 @@ class BusLocation: NSObject, NSCoding {
     var tripID: Int = 0
     var vehicleID: Int = 0
     
+    var _iconView: UIView? = nil
```

Comment:
why does this have _?

Reply:
I don't want it to be accessed. That's the syntax for "hidden" variables. I should just make it private